addons/mail/models/discuss/mail_guest.py

```diff
@@ -93,8 +93,8 @@ def _update_name(self, name):
             raise UserError(_("Guest's name is too long."))
         self.name = name
         for channel in self.channel_ids:
-            Store(bus_channel=channel).add(self, [*self._get_store_avatar_fields(), "name"]).bus_send()
-        Store(bus_channel=self).add(self, [*self._get_store_avatar_fields(), "name"]).bus_send()
+            Store(bus_channel=channel).add(self, "_store_avatar_fields").bus_send()
+        Store(bus_channel=self).add(self, "_store_avatar_fields").bus_send()
```

Comment:
Just wondering, but since we send on `self.channel_ids`, the guest will probably receive it. It looks like sending on the guest channel is redundant.